inc/fluent_bit/error_handler/namespace.php

```diff
@@ -0,0 +1,112 @@
+<?php
```

Comment:
Could we move this out of the fluent_bit namespace, and then provide Cloud APIs that are fluent agnostic, thinking that would be better decoupling. I should have do that with the CloudWatch logs too probably!

Reply:
Can do!